als-server/js/node-package/package.json

```diff
@@ -15,7 +15,7 @@
     "build:dist": "webpack --config ./webpack.config.js"
   },
   "dependencies": {
-    "ajv": "^7.2.4",
+    "ajv": "6.5.2",
```

Comment:
@agusayerza, you might try bumping `ajv` to latest minor version (6.12.6) if you want to mitigate a vulnerability (if it's regarding [CVE-2020-15366](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2020-15366)). I am running it without any issues but might need some testing.

I've reported the same issue before in https://github.com/aml-org/amf/issues/1037